src/Nethermind/Chains/baseline.json

```diff
@@ -36,7 +36,6 @@
         "eip2315Transition": "0x0",
         "eip2537Transition": "0x0",
         "eip2565Transition": "0x0",
-        "eip2718Transition": "0x0",
```

Comment:
you cannot remove it, otherwise a network without 2718 would accept 2718 transactions to the pool and add them to the block

Reply:
2718 is meaningless by itself. We need checks for each TxType so 2930 for now. This wasn't even used anywhere.

Reply:
how about unknown types?